public/app/features/dashboard/export_data/export_data_modal.ts

```diff
@@ -14,9 +15,9 @@ export class ExportDataModalCtrl {
       fileExport.exportTableDataToCsv(this.data, this.excel);
     } else {
       if (this.asRows) {
-        fileExport.exportSeriesListToCsv(this.data, this.dateTimeFormat, this.excel);
+        fileExport.exportSeriesListToCsv(this.data, this.dateTimeFormat, this.excel, this.utc);
       } else {
-        fileExport.exportSeriesListToCsvColumns(this.data, this.dateTimeFormat, this.excel);
+        fileExport.exportSeriesListToCsvColumns(this.data, this.dateTimeFormat, this.excel, this.utc);
```

Comment:
not sure isUtc is the best parameter name, we might want to add support for many different timezones, so sending a timezone paramater is probably more future proof ('utc', 'local', etc)